src/main/kotlin/gg/skytils/skytilsmod/core/Config.kt

```diff
@@ -1236,12 +1236,14 @@ object Config : Vigilant(
     )
     var blockZapperFatigueTimer = false
 
-    @Property(
-        type = PropertyType.SWITCH, name = "Compact Item Stars",
-        description = "Shortens item names with stars in them.",
-        category = "Miscellaneous", subcategory = "Items"
+       @Property(
+        type = PropertyType.SELECTOR, name = "Item Stars",
```

Comment:
change name to be like Item Stars Display 

also its supposed to be alphabetical